### PR TITLE
Logo and navbar changes

### PR DESCRIPTION
**Pull request by Shruti1462**

### CODE REVIEW

File: package-lock.json
Comment (bansaldhruv2779) [inline package-lock.json:1]: Don't push package json files

File: package.json
Comment (bansaldhruv2779) [inline package.json:1]: Don't push package json files

File: src/App.js
Comment (bansaldhruv2779) [inline src/App.js:1]: why you created this file?

File: src/index.js [hunk 1]
```diff
@@ -7,12 +7,19 @@ import {PersistGate} from "redux-persist/integration/react";
...
 import reportWebVitals from "./reportWebVitals";
 import Router from "./router";
-
+import {createBrowserRouter, RouterProvider} from "react-router-dom";
 import "@fontsource/roboto/300.css";
 import "@fontsource/roboto/400.css";
 import "@fontsource/roboto/500.css";
 import "@fontsource/roboto/700.css";
 
+const router = createBrowserRouter([
```
Comment (bansaldhruv2779) [inline src/index.js:16]: why this is needed?

Comment (Shruti1462) [reply src/index.js:16]: the changes were not showing in the navbar I checked it . It works without that .So I will remove this too.

**Comment by bansaldhruv2779** [comment]
Merged this change to master

**Pull request closed without merging**